CI error in unix gpu test_quantization_gpu.test_quantized_conv

more details

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-gpu/detail/PR-16067/1/pipeline
```
======================================================================

FAIL: test_quantization_gpu.test_quantized_conv

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/local/lib/python3.5/dist-packages/nose/case.py", line 198, in runTest

    self.test(*self.arg)

  File "/usr/local/lib/python3.5/dist-packages/nose/util.py", line 620, in newfunc

    return func(*arg, **kw)

  File "/work/mxnet/tests/python/quantization_gpu/../quantization/common.py", line 177, in test_new

    orig_test(*args, **kwargs)

  File "/work/mxnet/tests/python/quantization_gpu/../quantization/test_quantization.py", line 276, in test_quantized_conv

    check_quantized_conv((3, 4, 28, 28), (3, 3), 128, (1, 1), (1, 1), True, qdtype)

  File "/work/mxnet/tests/python/quantization_gpu/../quantization/test_quantization.py", line 268, in check_quantized_conv

    assert_almost_equal(output.asnumpy(), qoutput.asnumpy())

  File "/work/mxnet/python/mxnet/test_utils.py", line 533, in assert_almost_equal

    raise AssertionError(msg)

AssertionError: 

Items are not equal:

Error 195312500000000000.000000 exceeds tolerance rtol=0.000010, atol=0.000000.  Location of maximum error:(0, 69, 15, 9), a=0.003906, b=0.000000

 a: array([[[[  27046.992   ,    5385.004   ,  -82728.      , ...,

            26549.004   ,   43285.004   ,    8958.003   ],

         [  44756.016   ,   91666.      ,  -50330.      , ...,...

 b: array([[[[  27047,    5385,  -82728, ...,   26549,   43285,    8958],

         [  44756,   91666,  -50330, ...,  -10572,  -20104,   -1051],

         [  26405,   21287,   -1259, ...,  -29531,  -12399,   -4999],...

-------------------- >> begin captured logging << --------------------

common: INFO: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=60481325 to reproduce.

--------------------- >> end captured logging << ---------------------
```

--- a/tests/python/quantization/test_quantization.py
+++ b/tests/python/quantization/test_quantization.py
@@ -265,7 +265,7 @@ def test_quantized_conv():
         qoutput, min_range, max_range = conv_exe_int8.forward()
 
         if no_bias:
-            assert_almost_equal(output.asnumpy(), qoutput.asnumpy())
+            assert_almost_equal(output.asnumpy(), qoutput.asnumpy(), atol = 1)
         else:
             # with adding bias, accuracy loss should not be greater than one
             diff = mx.nd.abs(output - qoutput.astype(output.dtype))

